src/fetch/payouts.py

```diff
@@ -139,8 +145,19 @@ def as_payouts(self) -> list[Transfer]:
         Isolating the logic of how solvers are paid out according to their
             execution costs, rewards and slippage
         """
-        # We do not handle overdraft scenario here!
-        assert not self.is_overdraft()
+        quote_reward_cow = self.quote_reward_cow
+        result = []
+        if quote_reward_cow > 0:
+            result.append(
+                Transfer(
+                    token=Token(COW_TOKEN_ADDRESS),
+                    recipient=self.reward_target,
+                    amount_wei=quote_reward_cow,
+                )
+            )
+        if self.is_overdraft():
```

Comment:
Is there a reason we changed from `assert not self.is)overdraft()` to this if-statement? (sorry for my ignorance here)

Reply:
Oh good, thanks for clarifying!